app/assets/stylesheets/spree/frontend/components/checkout/payments/_gateway_payment.scss

```diff
@@ -0,0 +1,11 @@
+.gateway-method {
```

Comment:
I don't get this component, is it really needed? what is a `.gateway-method` and why it's in a file named `.gateway_payment`?